.github/workflows/test_live_variant_on_custom_liveview_project.yml

```diff
@@ -0,0 +1,69 @@
+name: "Test Live variant on custom LiveView project"
+
+on: push
+
+env:
+  OTP_VERSION: 23.1.1
+  ELIXIR_VERSION: 1.11.1
+  PHOENIX_VERSION: 1.5.5
```

Comment:
`test_variant_on_custom_project.yml` and `test_variant_on_standard_project.yml` use different versions:

```
OTP_VERSION: 23.1.4
ELIXIR_VERSION: 1.11.2
PHOENIX_VERSION: 1.5.7
````

Any reasons? If not then it should be synced with these versions.

Reply:
yeah, good catch, the new version is merged before this branch and I forget to adjust this branch.

Reply:
Upgraded in https://github.com/nimblehq/elixir-templates/pull/60/commits/6f3ec24e5493d82b65e13424b8f3a120a11e861d